x-pack/plugin/inference/src/main/java/org/elasticsearch/xpack/inference/InferenceIndex.java

```diff
@@ -62,7 +62,7 @@ public static XContentBuilder mappings() {
             return jsonBuilder().startObject()
                 .startObject(SINGLE_MAPPING_NAME)
                 .startObject("_meta")
-                .field("version", Version.CURRENT)
+                .field("version", BWC_MAPPINGS_VERSION)
```

Comment:
@davidkyle @droberts195 we need this change for both indices correct?

Reply:
Yes it needs to be removed from both indices but on reflection I think the adding `version` to the `_meta` mapping object is not necessary. 

Changes to the index mapping will be indicated by incrementing the `SystemIndexDescriptor.VERSION_META_KEY` value and there is no BWC concerns as this index does not exist before 8.11. There is no code that reads the `version` that I can find and in future we can rely on `SystemIndexDescriptor.VERSION_META_KEY`

Reply:
Ah ok, should I remove `version` from both the mappings then?

Reply:
Ok I removed the `version` field.